Error: Cannot find module 'babelify'

Our `browserify_execution.log` generates the following error traceback, after an initial `vagrant up`:

```bash
+ read -r file
Setting up watches.
Watches established.
...
+ filename=select_session
+ browserify /vagrant/src/jsx/select_session.jsx -t '[' babelify --presets es2015,react ']' -o /vagrant/src/js/select_session.js
Error: Cannot find module 'babelify' from '/vagrant/puppet/scripts'
    at /usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:46:17
    at process (/usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:173:43)
    at ondir (/usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:188:17)
    at load (/usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (/usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at /usr/lib/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:108:15)
```

--- a/puppet/scripts/browserify
+++ b/puppet/scripts/browserify
@@ -19,5 +19,5 @@ inotifywait /vagrant/src/jsx/ -m -e close_write -e move --format %f |
     filename="${file%.*}"
 
     # convert with 'browserify'
-    browserify /vagrant/src/jsx/"$file" -t [ babelify --presets es2015,react ] -o /vagrant/src/js/"$filename".js
+    (cd /usr/lib/node_modules && browserify /vagrant/src/jsx/"$file" -t [ babelify --presets es2015,react ] -o /vagrant/src/js/"$filename".js)
   done

